build-tools/packages/build-cli/src/commands/release/report-unreleased.ts

```diff
@@ -154,19 +153,30 @@ async function updateReportVersions(
 	version: string,
 	log: Logger,
 ): Promise<void> {
+	const clientPackage = "@fluidframework/aqueduct";
+	let clientVersionCaret;
+	let clientVersionSimple;
+
+	// eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
+	if (report[clientPackage]) {
+		clientVersionCaret = report[clientPackage].ranges.caret;
+		clientVersionSimple = report[clientPackage].version;
+	}
+
+	log.log(`Caret version: ${clientVersionCaret}`);
+	log.log(`Simple version: ${clientVersionSimple}`);
+
 	for (const packageName of Object.keys(report)) {
 		if (ignorePackageList.has(packageName)) {
 			continue;
 		}
 
-		// updates caret ranges
-		if (isInternalVersionRange(report[packageName].ranges.caret, true)) {
-			// If the caret range is a range, reset it to an exact version.
-			// Note: Post 2.0 release, the versions will no longer be internal versions so another condition will be required that will work after 2.0.
+		// todo: add better checks
+		if (report[packageName].ranges.caret === clientVersionCaret) {
```

Comment:
not sure if there are better ways to check in-repo 2.0+ versions.

Reply:
What's the goal of this check?

Reply:
The goal is to update the version for packages currently at version 2.1.0, which is the in-repo version for client packages. I guess the functions `isInternalVersionRange` and `isInternalVersionScheme` are not valid for versions 2.0 and above? 

Is there any better way to check this?